src/Stryker.Core/Stryker.Core/Compiling/FsharpCompilingProcess.cs

```diff
@@ -0,0 +1,97 @@
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.IO.Abstractions;
+using System.Linq;
+using FSharp.Compiler.SourceCodeServices;
+using Microsoft.CodeAnalysis.Emit;
+using Microsoft.Extensions.Logging;
+using Microsoft.FSharp.Collections;
+using Microsoft.FSharp.Control;
+using Stryker.Core.Exceptions;
+using Stryker.Core.Initialisation.Buildalyzer;
+using Stryker.Core.Logging;
+using Stryker.Core.MutationTest;
+using ParsedInput = FSharp.Compiler.SyntaxTree.ParsedInput;
+
+namespace Stryker.Core.Compiling
+{
+    public class FsharpCompilingProcess
+    {
+        private readonly MutationTestInput _input;
+        private readonly ILogger _logger;
+        private readonly IFileSystem _fileSystem;
+
+        public FsharpCompilingProcess(MutationTestInput input,
+            IRollbackProcess rollbackProcess, IFileSystem fileSystem)
+        {
+            _input = input;
+            _logger = ApplicationLogging.LoggerFactory.CreateLogger<FsharpCompilingProcess>();
+            _fileSystem = fileSystem;
```

Comment:
Minor nitpick but this doesn't follow our standard for dependencies.

Reply:
We use both throughout the code base